vimcat

```diff
@@ -6,13 +6,14 @@
 : if 0
     link="$0"
 
-    while true; do
+    while [ -h "$link" ]; do
         ls0="`ls -l \"$link\"`"
         new_link="`expr \"$ls0\" : '.* -> \(.*\)$'`"
-
-        [ -z "$new_link" ] && break
-
-        link="$new_link"
+        if expr "$new_link" : '/.*' > /dev/null; then
+            link="$new_link"
+        else
+            link="$(dirname $link)/$new_link"
```

Comment:
Please do not use POSIX command substitution at this point (before sourcing `prologue.sh`). Only backticks. And please quote `$link`.